lib/aqua/Taskfile.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+
+tasks:
+  install:
+    desc: Installs the Aqua CLI using the universal installer script, sets in PATH, and upgrades.
+    silent: true
+    cmds:
+      - |
+        curl -sSfL -O https://raw.githubusercontent.com/aquaproj/aqua-installer/v2.1.1/aqua-installer
+        echo "c2af02bdd15da6794f9c98db40332c804224930212f553a805425441f8331665  aqua-installer" | sha256sum -c
+        chmod +x aqua-installer
+        ./aqua-installer
+
+        printf "\n\n🌊 Aqua CLI has been installed, please add it to your PATH in ~/.bashrc or similar using the following command:"
+        printf '\n\nexport PATH=\"${AQUA_ROOT_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/aquaproj-aqua}/bin:$PATH\"\n\n'
+
+        rm ./aqua-installer
```

Comment:
I like the idea of doing cleanup with a keyword (see [`defer` docs](https://taskfile.dev/usage/#doing-task-cleanup-with-defer)). This way we ensure that cleanup will be performed even if some command in cmds fails.

Reply:
Great call -- Will do.